alicloud/resource_alicloud_sag_client_user.go

```diff
@@ -0,0 +1,150 @@
+package alicloud
+
+import (
+	"fmt"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/smartag"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func resourceAlicloudSagClientUser() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudSagClientUserCreate,
+		Read:   resourceAlicloudSagClientUserRead,
+		Update: resourceAlicloudSagClientUserUpdate,
+		Delete: resourceAlicloudSagClientUserDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"sag_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"band_width": {
+				Type:     schema.TypeInt,
+				Required: true,
+			},
+			"user_mail": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"client_ip": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
```

Comment:
Please keep schema parameters sync with API parameters. I think there is missing parameter `password`.